components/playback/controls/player/ButtonWidget.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div>
+    <div class="div__button-background" :style="background_cssprops"></div>
+    <span
+      v-for="btn_index in num_of_btn"
+      :key="btn_index"
+      :ref="btn_index.toString()"
+      class="span__button_label"
+      :style="btn_stateprop(btn_index)"
+      @click="selected(btn_index)"
+      @mouseenter="hover_active(btn_index)"
+      @mouseleave="hover_inactive(btn_index)"
+      >{{ button_names[btn_index - 1] }}
+    </span>
+    <div v-if="num_of_verticalline >= 1">
+      <canvas
+        v-for="line_index in num_of_verticalline"
+        :key="line_index"
+        class="canvas__vertical-line"
+        :style="btn_divider_display(line_index)"
+      >
+      </canvas>
+    </div>
+    <canvas
+      class="canvas__common-horizontal-line"
+      :style="'width: ' + (button_widget_width - 10) + 'px;'"
+    >
+    </canvas>
+  </div>
+</template>
+<script>
+export default {
+  name: "ButtonWidget",
+  props: {
+    button_names: {
+      type: Array,
+      required: true,
+    },
+    enabled_color: { type: String, default: "" },
```

Comment:
shouldn't there be provided default values for the enabled and disabled colors (matching the standard spec Ari uses)?  I thought this was mentioned in the Story requirements.

Currently, if I do `npm run dev` and don't pass in a color, the buttons show up as very light grey even when they are enabled.

Please create a visual test where props are not provided for the `enabled_color` or `disabled_color` but they render correctly using the standard color scheme Ari has been using in Mockflow
![image](https://user-images.githubusercontent.com/5436635/96633180-c150ad00-12e6-11eb-81bd-d95fdc2122b0.png)

Reply:
Fixed with a basic default colour values for now.
enabled_color: "#FFFFFF"
disabled_color: "#3F3F3F"

Reply:
Two JEST testcase are present to validate 
a) enabled_color prop is not present (is_enabled is true) then default value of #FFFFFF is rendered
b) disabled_color prop is not present (is_enabled is false) then default value of #3F3F3F is rendered.